install.rst

```diff
@@ -213,7 +213,7 @@ You can build PostgREST from source with `Stack <https://github.com/commercialha
   =====================  =======================================
   Ubuntu/Debian          libpq-dev, libgmp-dev, zlib1g-dev
   CentOS/Fedora/Red Hat  postgresql-devel, zlib-devel, gmp-devel
-  BSD                    postgresql95-client
+  BSD                    postgresql12-client
```

Comment:
Why not `postgresql13-client`, which is the latest released version?

(note, I have no experience with BSD, so maybe I'm missing something obvious)

Reply:
Ah, you already said that... because of:
https://cgit.freebsd.org/ports/tree/Mk/bsd.default-versions.mk#n109

I see...